services/monitoring/grafana/terraform/dashboards.tf

```diff
@@ -0,0 +1,35 @@
+// Import subfolders using an external script
+data "external" "subfolders" {
+  program = ["bash", "${path.module}/../../../../scripts/tf_helper_list_subfolders.sh", "${path.module}/../assets/shared/dashboards"]
```

Comment:
idea :bulb: 

since grafana dashboards are now controlled by terraform, we can store them inside terraform folder

Reply:
The two terraforms are better regared as completely seperate from each other, as in the "CD flow" they come in completely different point-of-times and have different expectations/dependencies. So for now I would suggest not to move and consolidate all terraform-files and -assets (*yet*?). Here I suggest to leave as-is for now and refactor later when it becomes more clear how our terraform(s) grow and interconnect.